src/iosrv/srv.pb-c.c

```diff
@@ -0,0 +1,105 @@
+/* Generated by the protocol buffer compiler.  DO NOT EDIT! */
+/* Generated from: srv.proto */
+
+/* Do not generate deprecated warnings for self */
+#ifndef PROTOBUF_C__NO_DEPRECATED
+#define PROTOBUF_C__NO_DEPRECATED
+#endif
+
+#include "srv.pb-c.h"
+void   srv__notify_ready_req__init
+                     (Srv__NotifyReadyReq         *message)
+{
+  static const Srv__NotifyReadyReq init_value = SRV__NOTIFY_READY_REQ__INIT;
+  *message = init_value;
+}
+size_t srv__notify_ready_req__get_packed_size
+                     (const Srv__NotifyReadyReq *message)
+{
+  assert(message->base.descriptor == &srv__notify_ready_req__descriptor);
+  return protobuf_c_message_get_packed_size ((const ProtobufCMessage*)(message));
```

Comment:
(style)  line over 80 characters

(style)  space prohibited between function name and open parenthesis '('